starport/services/chain/serve.go

```diff
@@ -97,7 +97,11 @@ func (c *Chain) Serve(ctx context.Context, options ...ServeOption) error {
 	}
 
 	// make sure that config.yml exists.
-	if _, err := conf.Locate(c.app.Path); err != nil {
+	if c.options.ConfigFile != "" {
+		if _, err := os.Stat(c.options.ConfigFile); err != nil {
```

Comment:
With https://github.com/tendermint/starport/pull/761#pullrequestreview-591864052 we don't need a change in this code block as well. It can just call `conf.Locate(c.options.ConfigFiles)`.